app/routes/ready-reckoner/selected-summary.js

```diff
@@ -13,12 +13,12 @@ const textMapping = {
   hedgerow: 'Hedgerows'
 }
 
-function getContentDetails (calculation, selectedStandards) {
-  let totalPayment = 0
+function getContentDetails (standardsSet, selectedStandards) {
+  const calculation = standardsSet.standards
+  const totalPayment = standardsSet.totalPayment
```

Comment:
Use value calculated from msg rather than re-calculating.

Reply:
The calculated value from the msg assumes all standards in the calculation have been selected by the user. The current code was a quick hack to generate the total based on the user selecting different standards without having to recalculate the total each time via the calculation engine. So, the current code will sometimes show the wrong value.

Given this version of the ready reckoner will change (and may never need this type of logic again), I don't think we need to revert to the previous code unless we want to leave it in a more consistent state.